README.md

```diff
@@ -1,60 +1,93 @@
 # observIQ OpenTelemetry Collector
 
+<center>
+
+[![Action Status](https://github.com/observIQ/observiq-otel-collector/workflows/Build/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions)
+[![Action Test Status](https://github.com/observIQ/observiq-otel-collector/workflows/Tests/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions)
+[![Go Report Card](https://goreportcard.com/badge/github.com/observIQ/observiq-otel-collector)](https://goreportcard.com/report/github.com/observIQ/observiq-otel-collector)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Gosec](https://github.com/observIQ/observiq-otel-collector/actions/workflows/gosec.yml/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions/workflows/gosec.yml)
+
+</center>
+
+## About
+
 The observIQ OpenTelemetry Collector is observIQ's distribution of the OpenTelemetry Collector.
```

Comment:
Links/references back to main Otel Collector/Otel & observIQ would look good here to me.

Reply:
Yeah Mike is going to give me a better version of this down the road. He said this is fine to ship for now.